beta-src/src/utils/formatTime.ts

```diff
@@ -0,0 +1,28 @@
+import { ParsedTime } from "../interfaces/ParsedTime";
+
+export default function formatTime(time: ParsedTime) {
+  let chipDisplay: string;
+  if (time.days) {
```

Comment:
it seems like you have a lot of duplicated logic in here. can you simply this conditional?